GVFS/GVFS.FunctionalTests/Settings.cs

```diff
@@ -30,23 +30,23 @@ public static void Initialize()
                 RepoToClone = @"https://gvfs.visualstudio.com/ci/_git/ForTests";
                 Commitish = @"FunctionalTests/20180214";
 
-                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
                 {
-                    EnlistmentRoot = @"C:\Repos\GVFSFunctionalTests\enlistment";
-                    PathToGVFS = @"GVFS.exe";
-                    PathToGit = @"C:\Program Files\Git\cmd\git.exe";
+                    EnlistmentRoot = @"C:\Repos\GVFSFunctionalTests\enlistment";
+                    PathToGVFS = @"GVFS.exe";
+                    PathToGit = @"C:\Program Files\Git\cmd\git.exe";
                     PathToBash = @"C:\Program Files\Git\bin\bash.exe";
-
+
                     ControlGitRepoRoot = @"C:\Repos\GVFSFunctionalTests\ControlRepo";
-                    FastFetchBaseRoot = @"C:\Repos\GVFSFunctionalTests\FastFetch";
+                    FastFetchBaseRoot = @"C:\Repos\GVFSFunctionalTests\FastFetch";
                     FastFetchRoot = Path.Combine(FastFetchBaseRoot, "test");
-                    FastFetchControl = Path.Combine(FastFetchBaseRoot, "control");
-                    PathToGVFSService = @"GVFS.Service.exe";
-                    BinaryFileNameExtension = ".exe";
-                }
-                else
+                    FastFetchControl = Path.Combine(FastFetchBaseRoot, "control");
+                    PathToGVFSService = @"GVFS.Service.exe";
+                    BinaryFileNameExtension = ".exe";
+                }
+                else

```

Comment:
There are a lot of whitespace (end-of-line?) changes here. Was the LF/CRLF situation here in a muddle before, and it's now homogeneous? Unfortunately, the GitHub diff doesn't show the nature of the newline characters.

Reply:
@pmj +1 to wishing we could see line endings on github!  The file was saved with CRLF which should be fixing any issues.  I think at some point we should consider updating all line endings in the repo and failing builds based on them changing.  Of course fixing all the line ends also pollutes history so not an easy decision.